---
title: "Project_Valerie_Florence_EpiGenetics"
author: "Valerie Heckendorn and Florence Marti"
date: "2025-05-07"
output: html_document
---

# Introduction and background

When monocytes—macrophage precursor cells found in the blood—migrate into tissue, they differentiate into macrophages and polarize into distinct functional phenotypes, most commonly referred to as either pro-inflammatory (M1) or anti-inflammatory (M2) macrophages. This polarization is primarily driven by the cytokine environment within the tissue. For example, interferon-γ (IFN-γ) and lipopolysaccharide (LPS) promote M1 polarization, while interleukin-4 (IL-4) and interleukin-13 (IL-13) favor M2 polarization. (Mantovani et al., 2002.)

Infection or vaccination can induce a phenomenon known as "trained immunity," where monocytes and macrophages undergo epigenetic reprogramming that enhances their long-term responsiveness to microbial stimuli (Netea et al., 2016).
On the other hand, during sepsis, monocytes can enter a state of immunoparalysis (also called endotoxin tolerance), a common complication of severe sepsis. In this state, monocytes exhibit a reduced capacity to produce pro-inflammatory cytokines (such as TNF-α and IL-6) in response to subsequent stimulation, while often maintaining or increasing anti-inflammatory cytokine production (such as IL-10) (Hotchkiss et al., 2013). Sepsis, and especially hospital acuired sepsis, is a life threatening condition which often arises due to ventilaor assisted pneumonia, bloostream infections from cathethers or intravenous lines, surgical site infections or urinary tract infections (Mary Garvey, 2024)

Histone and DNA modifications have been hypothesized to play a role in regulating monocyte and macrophage phenotypes (16), but data are limited to a few genes and histone modifications.
In the original publication, the authors compared histone modifications of LPS-primed monocytes mimicking sepsis and ß-glucan (BG)-primed monocytes, mimicking trained immunity. In our project, we aim to compare the histone marks of LPS- and BG-primed macrophages. Like in the original publication, we analysed the histone marks **H3K4me3** (promoters), **H3K4me1** (enhancers), and **M3K27ac** (active promoters and enhancers), and identified general changes in promoter and enhancer regions. We defined a set of *x* transcription, that were differentially expressed in the two groups and used these regions to determine if we could assign the LPS- or BG-primed macrophages to pro- or anti-inflammatory macrophage subgroups. 
Furthermore we looked at the combination of histone marks to see promoter and enhancer regions that are active, which we identified as a combination of the histone methylation and the acetlyation. 


## Experimental Setup
Monocytes were pre-incubated either with cell culture medium (RPMI), β-glucan (5µg/mL) or with LPS (100ng/mL), for 24 hours in a total volume of 10 mL. After a wash-out, cells were cultured in RPMI supplemented with 10% human pool serum. Monocytes were collected at different time points (0 h and 6 d after treatment) and counted before further treatment for chromatin immunoprecipitation, RNA or DNaseI treatment. Different donor Buffycoats (BC) were used as independent replicates. Replicates were generated for all the profiles including ChIPseq,RNAseq and DNaseIseq.

# Materials and Methods

For the analyis comparing the changes in histone marks upon priming with either BG or LPS, the histone marks of the primed macrophages are compared to the controls, cultured in RPMI. 
The filtered and preprocessed data is available to download (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE58310), for the histone modifications, two different BC donors were available, BC8 and BC17. We chose to do the analysis using BC17 due to time and data storage restrictions. 

Furthermore, the authors provided a list of human transcription factors (TFs), since this list contains 1581 human TFs, we created a more cohesive list of transcription factors to find an answer to our question whether the BG and LPS primed macrophages can be assigned to pro- or anti-inflammatory macrophage subgroups. For this, we created a list of 26 transcription factors that are known to be important in pro-inflammatory macrophages, and 15 transcription factors that are important in anti-inflammatory macrophages. The list was generated using artificial intelligence and cross referencing the sources provided (Li et al. 2018)


For the analysis we used both the .bed files, as well as the .wig files. Converting the .wig to .bw files was done using the terminal and with the help of ChatGPT in the following manner: 

```{r, eval = FALSE}
The default interactive shell is now zsh.
To update your account to use zsh, please run `chsh -s /bin/zsh`.
For more details, please visit https://support.apple.com/kb/HT208050.
Florence-Marti:~ florencemarti$ cd "/Users/florencemarti/Documents/GitHub/EpiGen_Project_Valerie_Florence/Histone Modifications BG"
Florence-Marti:Histone Modifications BG florencemarti$ cd BigWig
Florence-Marti:BigWig florencemarti$ curl -O https://hgdownload.soe.ucsc.edu/admin/exe/macOSX.x86_64/wigToBigWig
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:--  0     0    0     0    0     0      0      0 --:--:-- --:--:--  0     0    0     0    0     0      0      0 --:--:--  0:00:01  0     0    0     0    0     0      0      0 --:--:--  0:00:02  1 8210k    1  117k    0     0  37835      0  0:03:42  0:00:03 24 8210k   24 2031k    0     0   486k      0  0:00:16  0:00:04 50 8210k   50 4109k    0     0   778k      0  0:00:10  0:00:05 76 8210k   76 6242k    0     0   997k      0  0:00:08  0:00:06 95 8210k   95 7828k    0     0  1082k      0  0:00:07  0:00:07100 8210k  100 8210k    0     0  1084k      0  0:00:07  0:00:07 --:--:-- 1837k
Florence-Marti:BigWig florencemarti$ chmod +x wigToBigWig
Florence-Marti:BigWig florencemarti$ curl -O https://hgdownload.soe.ucsc.edu/goldenPath/hg19/bigZips/hg19.chrom.sizes
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:--100  1971  100  1971    0     0   3988      0 --:--:-- --:--:-- --:--:--  3997
Florence-Marti:BigWig florencemarti$ nano convert_wigs.sh
Florence-Marti:BigWig florencemarti$ chmod +x convert_wigs.sh
Florence-Marti:BigWig florencemarti$ ./convert_wigs.sh
Converting ./wig_files/H3K27ac_BG.wig to ./bigwig_files/H3K27ac_BG.bw
Converting ./wig_files/H3K27ac_M0.wig to ./bigwig_files/H3K27ac_M0.bw
line 22203217 of ./wig_files/H3K27ac_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195220
line 22203218 of ./wig_files/H3K27ac_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195230
line 22203219 of ./wig_files/H3K27ac_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195240
line 22203220 of ./wig_files/H3K27ac_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195250
line 22203221 of ./wig_files/H3K27ac_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195260
line 22203222 of ./wig_files/H3K27ac_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195270
line 22203223 of ./wig_files/H3K27ac_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195280
line 22203224 of ./wig_files/H3K27ac_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195290
line 22203225 of ./wig_files/H3K27ac_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195300
line 22203226 of ./wig_files/H3K27ac_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195310
line 22203227 of ./wig_files/H3K27ac_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195320
line 22203228 of ./wig_files/H3K27ac_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195330
line 22203229 of ./wig_files/H3K27ac_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195340
Converting ./wig_files/H3K27ac_MF.wig to ./bigwig_files/H3K27ac_MF.bw
Converting ./wig_files/H3K4me1_BG.wig to ./bigwig_files/H3K4me1_BG.bw
Converting ./wig_files/H3K4me1_M0.wig to ./bigwig_files/H3K4me1_M0.bw
line 37037655 of ./wig_files/H3K4me1_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195220
line 37037656 of ./wig_files/H3K4me1_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195230
line 37037657 of ./wig_files/H3K4me1_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195240
line 37037658 of ./wig_files/H3K4me1_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195250
line 37037659 of ./wig_files/H3K4me1_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195260
line 37037660 of ./wig_files/H3K4me1_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195270
line 37037661 of ./wig_files/H3K4me1_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195280
line 37037662 of ./wig_files/H3K4me1_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195290
line 37037663 of ./wig_files/H3K4me1_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195300
line 37037664 of ./wig_files/H3K4me1_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195310
line 37037665 of ./wig_files/H3K4me1_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195320
line 37037666 of ./wig_files/H3K4me1_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195330
line 37037667 of ./wig_files/H3K4me1_M0.wig: chromosome chr17 has 81195210 bases, but item ends at 81195340
Converting ./wig_files/H3K4me1_MF.wig to ./bigwig_files/H3K4me1_MF.bw
line 29484841 of ./wig_files/H3K4me1_MF.wig: chromosome chr17 has 81195210 bases, but item ends at 81195220
line 29484842 of ./wig_files/H3K4me1_MF.wig: chromosome chr17 has 81195210 bases, but item ends at 81195230
line 29484843 of ./wig_files/H3K4me1_MF.wig: chromosome chr17 has 81195210 bases, but item ends at 81195240
line 29484844 of ./wig_files/H3K4me1_MF.wig: chromosome chr17 has 81195210 bases, but item ends at 81195250
line 29484845 of ./wig_files/H3K4me1_MF.wig: chromosome chr17 has 81195210 bases, but item ends at 81195260
line 29484846 of ./wig_files/H3K4me1_MF.wig: chromosome chr17 has 81195210 bases, but item ends at 81195270
line 29484847 of ./wig_files/H3K4me1_MF.wig: chromosome chr17 has 81195210 bases, but item ends at 81195280
line 29484848 of ./wig_files/H3K4me1_MF.wig: chromosome chr17 has 81195210 bases, but item ends at 81195290
line 29484849 of ./wig_files/H3K4me1_MF.wig: chromosome chr17 has 81195210 bases, but item ends at 81195300
line 29484850 of ./wig_files/H3K4me1_MF.wig: chromosome chr17 has 81195210 bases, but item ends at 81195310
line 29484851 of ./wig_files/H3K4me1_MF.wig: chromosome chr17 has 81195210 bases, but item ends at 81195320
line 29484852 of ./wig_files/H3K4me1_MF.wig: chromosome chr17 has 81195210 bases, but item ends at 81195330
line 29484853 of ./wig_files/H3K4me1_MF.wig: chromosome chr17 has 81195210 bases, but item ends at 81195340
Converting ./wig_files/H3K4me3_BG.wig to ./bigwig_files/H3K4me3_BG.bw
Converting ./wig_files/H3K4me3_M0.wig to ./bigwig_files/H3K4me3_M0.bw
Converting ./wig_files/H3K4me3_MF.wig to ./bigwig_files/H3K4me3_MF.bw
Florence-Marti:BigWig florencemarti$ 

```


# Setup and download
```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
setwd("~/Documents/GitHub/EpiGen_Project_Valerie_Florence")
```

```{r, include = FALSE}
suppressPackageStartupMessages({
  library(AnnotationHub)
  library(ensembldb)
  library(GenomicRanges)
  library(epiwraps)
  library(rtracklayer)
  library(ggplot2)
  library(magick)
  library(R.utils) 
  library(MASS)
})
```



```{r, eval = FALSE, include = FALSE}
#Downloading and unzipping the four files 
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE58310&format=file&file=GSE58310%5FBGall4Buffy891112merge%2Ehg19%2Ehotspot%2Etwopass%2Efdr0%2E01%2Ebed%2Egz","BG.bed.gz")

gunzip("BG.bed.gz")
BG <- rtracklayer::import("BG.bed", format = "BED")

download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE58310&format=file&file=GSE58310%5FBuffy101112Day0merge%2Ehg19%2Ehotspot%2Etwopass%2Efdr0%2E01%2Ebed%2Egz", "buffy_hotspot.bed.gz")
gunzip("buffy_hotspot.bed.gz")
independent_replicated<- rtracklayer::import("buffy_hotspot.bed", format = "BED")

download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE58310&format=file&file=GSE58310%5FLPSall4Buffy891112merge%2Ehg19%2Ehotspot%2Etwopass%2Efdr0%2E01%2Ebed%2Egz", "LPS.bed.gz")
gunzip("LPS.bed.gz")
LPS <- rtracklayer::import("LPS.bed", format = "BED")

download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE58310&format=file&file=GSE58310%5FRPMIall4Buffy891112merge%2Ehg19%2Ehotspot%2Etwopass%2Efdr0%2E01%2Ebed%2Egz", "RPMI.bed.gz")
gunzip("RPMI.bed.gz")
control <- rtracklayer::import("RPMI.bed", format = "BED")

download.file("https://www.ncbi.nlm.nih.gov/geo/download/?format=file&type=rnaseq_counts&file=Human.GRCh38.p13.annot.tsv.gz", "GRCh38annot.tsv.gz")
gunzip("GRCh38annot.tsv.gz")

download.file("https://ftp.ncbi.nlm.nih.gov/geo/series/GSE58nnn/GSE58310/suppl/GSE58310_GeneExpression.csv.gz","gene_expression.csv.gz")

gunzip("gene_expression.csv.gz")
gene_expression <- read.csv("gene_expression.csv")

colnames(gene_expression) <- c("Gene_Id","Control_D0_1", "Control_D0_2", "Control_D0_3","Control_D0_4", "Control_D6_1", "Control_D6_2", "Control_D6_3","Control_D6_4", "LPS_D6_1", "LPS_D6_2", "LPS_D6_3","LPS_D6_4", "BG_D6_1", "BG_D6_2", "BG_D6_3","BG_D6_4")
```

## Downloading the histone modifications

### Monocytes
```{r}
options(timeout = 12000)

#H3K4me3
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSM1406294&format=file&file=GSM1406294%5FMo%5FH3K4me3%5F5631%5FNR%2Ebed%2Egz", "H3K4me3_M0_BC17.bed.gz")
gunzip("H3K4me3_M0_BC17.bed.gz")
#H3K4me3_M0_BC17 <- rtracklayer::import("H3K4me3_M0_BC17.bed", format = "BED")


#H3K27ac
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSM1406295&format=file&file=GSM1406295%5FMo%5FH3K27ac%5F5632%5FNR%2Ebed%2Egz", "H3K27ac_M0_BC17.bed.gz")
gunzip("H3K27ac_M0_BC17.bed.gz")
#H3K27ac_M0_BC17 <- rtracklayer::import("H3K27ac_M0_BC17.bed", format = "BED")


#H3K3me1
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSM1406296&format=file&file=GSM1406296%5FMo%5FH3K4me1%5F5633%5FNR%2Ebed%2Egz", "H3K4me1_M0_BC17.bed.gz")
gunzip("H3K4me1_M0_BC17.bed.gz")
#H3K4me1_M0_BC17 <- rtracklayer::import("H3K4me1_M0_BC17.bed", format = "BED")


```
### Macrophages

```{r}
options(timeout = 12000)

#H3K4me3
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSM1406297&format=file&file=GSM1406297%5FMf%5FH3K4me3%5F5634%5FNR%2Ebed%2Egz", "H3K4me3_MF_BC17.bed.gz")
gunzip("H3K4me3_MF_BC17.bed.gz")
#H3K4me3_MF_BC17 <- rtracklayer::import("H3K4me3_MF_BC17.bed", format = "BED")


#H3K27ac
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSM1406298&format=file&file=GSM1406298%5FMf%5FH3K27ac%5F5635%5FNR%2Ebed%2Egz", "H3K27ac_MF_BC17.bed.gz")
gunzip("H3K27ac_MF_BC17.bed.gz")
#H3K27ac_MF_BC17 <- rtracklayer::import("H3K27ac_MF_BC17.bed", format = "BED")


#H3K3me1
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSM1406299&format=file&file=GSM1406299%5FMf%5FH3K4me1%5F5636%5FNR%2Ebed%2Egz", "H3K4me1_MF_BC17.bed.gz")
gunzip("H3K4me1_MF_BC17.bed.gz")
#H3K4me1_MF_BC17 <- rtracklayer::import("H3K4me1_MF_BC17.bed", format = "BED")

```

### LPS primed

```{r}

```

### BG primed
```{r}
options(timeout = 12000)

#H3K4me3
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSM1406303&format=file&file=GSM1406303%5FBG%5FH3K4me3%5F5640%5FNR%2Ebed%2Egz", "H3K4me3_BG_BC17.bed.gz")
gunzip("H3K4me3_BG_BC17.bed.gz")
#H3K4me3_BG_BC17 <- rtracklayer::import("H3K4me3_BG_BC17.bed", format = "BED")


#H3K27ac
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSM1406304&format=file&file=GSM1406304%5FBG%5FH3K27ac%5F5641%5FNR%2Ebed%2Egz", "H3K27ac_BG_BC17.bed.gz")
gunzip("H3K27ac_BG_BC17.bed.gz")
#H3K27ac_BG_BC17 <- rtracklayer::import("H3K27ac_BG_BC17.bed", format = "BED")


#H3K3me1
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSM1406305&format=file&file=GSM1406305%5FBG%5FH3K4me1%5F5642%5FNR%2Ebed%2Egz", "H3K4me1_BG_BC17.bed.gz")
gunzip("H3K4me1_BG_BC17.bed.gz")
#H3K4me1_BG_BC17 <- rtracklayer::import("H3K4me1_BG_BC17.bed", format = "BED")

```
# Downloading bigwig files


### Monocytes
```{r}
options(timeout = 12000)

#H3K4me3
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSM1406294&format=file&file=GSM1406294%5FMo%5FH3K4me3%5F5631%5FNR%2Ewig%2Egz", "H3K4me3_M0.wig.gz")
gunzip("H3K4me3_M0.wig.gz")
#H3K4me3_M0_BC17 <- rtracklayer::import("H3K4me3_M0_BC17.bed", format = "BED")


#H3K27ac
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSM1406295&format=file&file=GSM1406295%5FMo%5FH3K27ac%5F5632%5FNR%2Ewig%2Egz", "H3K27ac_M0.wig.gz")
gunzip("H3K27ac_M0.wig.gz")
#H3K27ac_M0_BC17 <- rtracklayer::import("H3K27ac_M0.wig", format = "BED")


#H3K3me1
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSM1406296&format=file&file=GSM1406296%5FMo%5FH3K4me1%5F5633%5FNR%2Ewig%2Egz", "H3K4me1_M0.wig.gz")
gunzip("H3K4me1_M0.wig.gz")
#H3K4me1_M0_BC17 <- rtracklayer::import("H3K4me1_M0.wig", format = "BED")


```
### Macrophages

```{r}
options(timeout = 12000)

#H3K4me3
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSM1406297&format=file&file=GSM1406297%5FMf%5FH3K4me3%5F5634%5FNR%2Ewig%2Egz", "H3K4me3_MF.wig.gz")
gunzip("H3K4me3_MF.wig.gz")
#H3K4me3_MF_BC17 <- rtracklayer::import("H3K4me3_MF_BC17.bed", format = "BED")


#H3K27ac
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSM1406298&format=file&file=GSM1406298%5FMf%5FH3K27ac%5F5635%5FNR%2Ewig%2Egz", "H3K27ac_MF.wig.gz")
gunzip("H3K27ac_MF.wig.gz")
#H3K27ac_MF_BC17 <- rtracklayer::import("H3K27ac_MF.wig", format = "BED")


#H3K3me1
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSM1406299&format=file&file=GSM1406299%5FMf%5FH3K4me1%5F5636%5FNR%2Ewig%2Egz", "H3K4me1_MF.wig.gz")
gunzip("H3K4me1_MF.wig.gz")
#H3K4me1_MF_BC17 <- rtracklayer::import("H3K4me1_MF.wig", format = "BED")

```

### LPS primed

```{r}

```

### BG primed
```{r}
options(timeout = 12000)

#H3K4me3
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSM1406303&format=file&file=GSM1406303%5FBG%5FH3K4me3%5F5640%5FNR%2Ewig%2Egz", "H3K4me3_BG.wig.gz")
gunzip("H3K4me3_BG.wig.gz")
#H3K4me3_BG_BC17 <- rtracklayer::import("H3K4me3_BG.wig", format = "BED")


#H3K27ac
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSM1406304&format=file&file=GSM1406304%5FBG%5FH3K27ac%5F5641%5FNR%2Ewig%2Egz", "H3K27ac_BG.wig.gz")
gunzip("H3K27ac_BG.wig.gz")
#H3K27ac_BG_BC17 <- rtracklayer::import("H3K27ac_BG.wig", format = "BED")


#H3K3me1
download.file("https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSM1406305&format=file&file=GSM1406305%5FBG%5FH3K4me1%5F5642%5FNR%2Ewig%2Egz", "H3K4me1_BG.wig.gz")
gunzip("H3K4me1_BG.wig.gz")
#H3K4me1_BG_BC17 <- rtracklayer::import("H3K4me1_BG.wig", format = "BED")

```
# Transcription Factors
```{r}
setwd("~/Documents/GitHub/EpiGen_Project_Valerie_Florence/Data")
```


### Looking at the data
```{r}
head(H3K4me1_BG_BC17)
head(H3K4me1_M0_BC17)
head(H3K4me1_MF_BC17)
# we have seqnames = chromosomes, the ranges and the strand, aswell as name column, which we interpret as the signal intensity. 

```


Epigenetic dynamics (H3K4me1, H3K4me3, H3K27ac, DNase I accessibility) during monocyte differentiation into macrophages.


Transcriptional networks and transcription factor (TF) motifs linked to immune training vs. tolerance.

## Bibliography

Mantovani A, Sozzani S, Locati M, Allavena P, Sica A. Macrophage polarization: tumor-associated macrophages as a paradigm for polarized M2 mononuclear phagocytes. Trends Immunol. 2002 Nov;23(11):549-55. doi: 10.1016/s1471-4906(02)02302-5. PMID: 12401408.

Netea MG, Joosten LA, Latz E, Mills KH, Natoli G, Stunnenberg HG, O'Neill LA, Xavier RJ. Trained immunity: A program of innate immune memory in health and disease. Science. 2016 Apr 22;352(6284):aaf1098. doi: 10.1126/science.aaf1098. Epub 2016 Apr 21. PMID: 27102489; PMCID: PMC5087274.

Hotchkiss RS, Moldawer LL, Opal SM, Reinhart K, Turnbull IR, Vincent JL. Sepsis and septic shock. Nat Rev Dis Primers. 2016 Jun 30;2:16045. doi: 10.1038/nrdp.2016.45. PMID: 28117397; PMCID: PMC5538252.

Garvey M. Hospital Acquired Sepsis, Disease Prevalence, and Recent Advances in Sepsis Mitigation. Pathogens. 2024 May 30;13(6):461. doi: 10.3390/pathogens13060461. PMID: 38921759; PMCID: PMC11206921.

Li H, Jiang T, Li MQ, Zheng XL, Zhao GJ. Transcriptional Regulation of Macrophages Polarization by MicroRNAs. Front Immunol. 2018 May 28;9:1175. doi: 10.3389/fimmu.2018.01175. PMID: 29892301; PMCID: PMC5985397.